tests/unit/test_spatial_methods/test_finite_volume/test_integration.py

```diff
@@ -16,142 +15,6 @@
 
 
 class TestFiniteVolumeIntegration:
-    def test_definite_integral(self):
```

Comment:
this test is not related to the main change in this PR, it should not be removed